lib/fluent/plugin/in_windows_eventlog.rb

```diff
@@ -21,12 +22,14 @@ class WindowsEventLogInput < Input
 
     config_param :tag, :string
     config_param :read_interval, :time, default: 2
-    config_param :pos_file, :string, default: nil
+    config_param :pos_file, :string, default: nil,
+                 obsoleted: "This section is not used anymore. Use 'store_pos' instead."
     config_param :channels, :array, default: ['Application']
     config_param :keys, :string, default: []
     config_param :read_from_head, :bool, default: false
     config_param :from_encoding, :string, default: nil
     config_param :encoding, :string, default: nil
+    config_param :store_pos, :bool, default: false
```

Comment:
`pos_file` is obsoleted, so this plugin doesn't work without specifying `store_pos true`.
So, this parameter doesn't have to exist (we can remove this parameter), and we suppose that this plugin always uses storage plugin to store positions.